src/components/atoms/Icon/Icon.css

```diff
@@ -0,0 +1,8 @@
+/* Highlighted styles */
+.icon-container.highlighted {
+  background-image: url('data:image/svg+xml,%3Csvg%20viewBox%3D%220%200%2049%2019%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Cellipse%20cx%3D%2224.5%22%20cy%3D%229.5%22%20rx%3D%2224.5%22%20ry%3D%229.5%22%20fill%3D%22%239FD5B3%22%2F%3E%3C%2Fsvg%3E');
+  background-repeat: no-repeat;
+  background-position-y: bottom;
+  background-size: 100% auto;
+  padding-bottom: 5px;
```

Comment:
<img width="77" alt="312214742-327132d6-9c4c-42cc-97ec-acbd89eff6fc" src="https://github.com/CityOfDetroit/COD-Design-System/assets/143553259/0b24ebf6-409b-4f21-882c-17de44908db7">

You'll notice that the smaller icons start lurching away from the ellipse. The reason is likely due to this padding being a constant pixel value instead of a relative value like a percentage that will scale with the container size.

I would try playing around with percentages for the bottom padding. Maybe start with 10% since the XL variant looks pretty good and that is 5px bottom padding with a 54 px height icon (5 / 54 = approx. 10%).

Reply:
10% was too much too. I did .3% and it looks pretty good

<img width="193" alt="Screenshot 2024-03-13 at 11 43 07 AM" src="https://github.com/CityOfDetroit/COD-Design-System/assets/141044304/40d1641b-57b2-4ee7-8f27-b14609aab22a">